sql/migrations/local/0001_initial.sql

```diff
@@ -18,5 +18,7 @@ CREATE TABLE initial_post
     post_indices  VARCHAR NOT NULL,
     post_pow      UNSIGNED LONG INT NOT NULL,
 
-    commit_atx    CHAR(32)
+    num_units     UNSIGNED INT NOT NULL,
+    commit_atx    CHAR(32) NOT NULL,
+    vrf_nonce     UNSIGNED LONG INT NOT NULL
```

Comment:
You didn't decide to include the labels per unit here. It makes sense as it is part of the network config (hence same for everybody, at least so far). On the other hand, it is used in the POST verification flow and is not required to be equal to the network config - it must be greater or equal (although it's unclear why anybody would use a bigger value).

We should probably either include labels_per_unit here to be consistent or start wiping out using labels_per_unit from POST validation (use the value from the config instead).

Reply:
once local database is released it is not possible to extend initial migration

Reply:
Labels per unit field is supposed to be removed from ATX: https://github.com/spacemeshos/go-spacemesh/issues/4767. So let's not add it here.

> once local database is released it is not possible to extend initial migration

I think @mafa wants to include these fields in the initial release of the local state.

Reply:
v1.3 is not released yet, or is it?

Reply:
it is not released, it has release candidate and branch already there. i don't understand though, @fasmat plans to cherry pick this change to that branch before it is released? thats something weird

Reply:
- I was not planning on including this change in `v1.3` but in the next version.
- I will update this to be its separate database migration and add labels per unit
- Removing labes per unit from this state and from ATXs is a bigger change that should be tackled in a dedicated PR that closes #4767